spring-data-rest-webmvc/src/main/java/org/springframework/data/rest/webmvc/mapping/AssociationLinks.java

```diff
@@ -99,4 +103,31 @@ public boolean isLinkableAssociation(PersistentProperty<?> property) {
 		metadata = mappings.getMappingFor(property.getActualType());
 		return metadata == null ? false : metadata.isExported();
 	}
+
+	/**
+	 * Returns whether the given property is an association that should be exported.
+	 *
+	 * @param property can be {@literal null}
+	 * @return
+	 */
+	public boolean isExportableAssociation(PersistentProperty<?> property) {
```

Comment:
I wonder whether we should have this as separate method. I haven't thought through this entirely yet but wouldn't a lot of clients call both `isExportableAssociation(…)` and `isLinkableAssociation(…)` in a single if-clause? Might be worth exploring if tests break if we simply augment the latter with the additional logic.